src/components/speakers/content/speakers.js

```diff
@@ -1,29 +1,20 @@
 export const speakersList = [
   {
-    name: "Ibekwe Laster",
-    title: "Software Engineer",
-    company: "Hooli",
-    introText: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-    image: "http://cdn02.cdn.justjaredjr.com/wp-content/uploads/headlines/2017/05/despicable-me-3-clip-minions-stage.jpg",
+    name: "Awesome Speaker",
+    title: "Front-end Engineer",
+    company: "Company",
+    introText: "All round nice guy :)",
+    image: "http://financialspectrum.com.au/wp-content/uploads/Headshot-Placeholder.png",
     githubUrl: "#",
     twitterUrl: "#",
```

Comment:
In the component that uses these urls, can we show the links to the profile url (github, twitter) only when it has actual links to the profiles?

Reply:
Yeah sure. Good point